src/Nri/Ui/Modal/V11.elm

```diff
@@ -319,13 +322,25 @@ warning =
         ]
 
 
+{-| Include the close button.
+-}
+closeButton : Attribute
+closeButton =
+    Attribute (\attrs -> { attrs | closeButton = True })
+
+
 {-| This is the default setting.
 -}
 showTitle : Attribute
 showTitle =
     Attribute (\attrs -> { attrs | visibleTitle = True })
 
 
+focusTrap : FocusTrap -> Attribute
```

Comment:
💭 @tesk9 how would you feel about making the focus trap a required part of the config instead of an attribute of the modal? I think I liked how the FocusManager forced me to think about focus management for the modal.

Reply:
I did it! Let me know if you disagree though, we can totally revert.